packages/expo-modules-core/android/src/main/java/expo/modules/core/BasePackage.java

```diff
@@ -35,6 +36,11 @@
     return Collections.emptyList();
   }
 
+  @Override
+  public int getPackagePriority() {
+    return 0;
+  }
```

Comment:
Is there any use case where we need the priority? Read my comment here https://github.com/expo/expo/pull/14867#issuecomment-950862729